README.md

```diff
@@ -307,10 +307,10 @@ notification should be emitted. We can fetch it like this:
 
 ```ruby
   to_be_called_on_close = -> () do
-    # Fnalize?
+    # Finalize?
```

Comment:
Hmm why didn't the spell-checker fix that? Who knows.
Thanks a lot! Now you are becoming the third contributor to Reacto